src/eva/vision/models/networks/from_timm.py

```diff
@@ -0,0 +1,36 @@
+"""Helper wrapper class for timm models."""
+
+from typing import Any, Dict
+
+import timm
+
+from eva.core.models.networks import wrappers
+
+
+class TimmModel(wrappers.ModelFromFunction):
```

Comment:
Would it make sense to move this to `from eva.vision.models.networks.wrappers` (since this is not a network architecture, but a wrapper for timm).

Reply:
I put it here because timm is a vision model wrapper and I think we install `timm`  only on `eva[vision]` 🤔  what do you think?

Reply:
Yes, that makes sense. I proposed to leave it in `eva.vision`, but creating a new `.wrappers` module there (which would be in line with the module hierarchy in core)

Reply:
maybe we can make an abstract onces we have another module?

Reply:
how do you mean that? we already have an abstract base class for model wrappers in (`eva.core.models.networks.wrappers.BaseModel`)

Reply:
I think that you recommend to have a new abstract vision one no?